src/components/create_cluster/index.js

```diff
@@ -378,88 +261,47 @@ class CreateCluster extends React.Component {
               </div>
             </div>
 
-            <div>
-              <div className='row section'>
-                <div className='col-12'>
-                  <h3 className='table-label'>Worker Node Configuration</h3>
-                  {this.props.provider === 'kvm' ? (
-                    <div className='checkbox'>
-                      <label htmlFor='syncWorkers'>
-                        <input
-                          type='checkbox'
-                          ref={i => {
-                            this.syncWorkers = i;
-                          }}
-                          id='syncWorkers'
-                          onChange={this.syncWorkersChanged}
-                          checked={this.state.syncWorkers}
-                        />
-                        Use same configuration for all worker nodes
-                      </label>
-                    </div>
-                  ) : (
-                    undefined
-                  )}
-                </div>
+            <div className='row section'>
+              <div className='col-3'>
+                <h3 className='table-label'>Node Count</h3>
               </div>
-              <div className='row'>
-                {this.state.workers.map((worker, index) => {
-                  if (this.props.provider === 'aws') {
-                    return (
-                      <NewAWSWorker
-                        key={'Worker ' + worker.id}
-                        allowedInstanceTypes={this.props.allowedInstanceTypes}
-                        worker={worker}
-                        index={index}
-                        readOnly={this.state.syncWorkers && index !== 0}
-                        deleteWorker={this.deleteWorker.bind(this, index)}
-                        onWorkerUpdated={this.updateWorker.bind(this, index)}
-                      />
-                    );
-                  } else if (this.props.provider === 'kvm') {
-                    return (
-                      <NewKVMWorker
-                        key={'Worker ' + worker.id}
-                        worker={worker}
-                        index={index}
-                        readOnly={this.state.syncWorkers && index !== 0}
-                        deleteWorker={this.deleteWorker.bind(this, index)}
-                        onWorkerUpdated={this.updateWorker.bind(this, index)}
+              <div className='col-9'>
+                <form
+                  onSubmit={e => {
+                    e.preventDefault();
+                  }}
+                >
+                  <div>
+                    <p>
+                      To have full control over cluster scaling, set both
```

Comment:
This makes me think.

I don't understand really what will happen here.

In a previous mockup, i saw some radio buttons, that was pretty clear to me.

The actual behaviour now is not so explicit to me. Perhaps the label needs to be improved.

ping @marians, since I hear from @xh3b4sd that this was your labelling suggestion. 

How do these numbers interact with the 'scale cluster' button? Should that be disabled when these numbers are not equal to eachother?

Reply:
The radio button design (modal switch between current manual scaling and new autoscaling) has been discarded, as it's actually simpler overall to have the autoscaler in place everywhere. Setting min equal to max can be seen as a trick to make the set a static cluster size.

My current status is that the scaling function as it is (modifying the nodes array length in the cluster details via PATCH) will no longer work from the release that introduces autoscaling. But team spirit has the latest news on this.

I am not aware that a final decision has been made regarding what this means for the scaling function in happa (and gsctl, either). Where and how should users access and manipulate the min/max limits for a cluster?

Reply:
I think we could just extend the scaling modal with min and max and go from there. 

<img width="1920" alt="screenshot 2018-12-05 at 15 04 34" src="https://user-images.githubusercontent.com/552769/49518509-17f1a100-f89f-11e8-963b-a6679bcd4dd2.png">

Reply:
I like having a separate endpoint for scaling, just like status. The scaling subresource in k8s is also a separate one.

Reply:
How about 'To disable autoscaling, set both numbers to the same value' as the description?

Reply:
Works for me

Reply:
Added the suggestion.